packages/common/src/context/comments/commentsContext.tsx

```diff
@@ -109,13 +119,30 @@ export function CommentSectionProvider<NavigationProp>(
     sortMethod: currentSort,
     userId: currentUserId
   })
+  console.log({ commentIds })
   const queryClient = useQueryClient()
   // hard refreshes all data
-  const reset = () => {
+  const resetComments = () => {
+    // Reset our comment count since we're reloading comments again - aka can hide the "new comments" button
+    resetPreviousCommentCount(queryClient, entityId)
     queryClient.resetQueries({ queryKey: [QUERY_KEYS.trackCommentList] })
     queryClient.resetQueries({ queryKey: [QUERY_KEYS.comment] })
     queryClient.resetQueries({ queryKey: [QUERY_KEYS.commentReplies] })
   }
+
+  const { data: commentCountData, isLoading: isCommentCountLoading } =
+    useTrackCommentCount(entityId, currentUserId, true)
+
+  console.log({ commentCountData })
+
+  const hasNewComments = useMemo(
```

Comment:
dont need to memoize this since its a fast compute

Reply:
ah good catch, I had this in here when I was debugging some infinite rerender issues